webviz_subsurface/_abbreviations/__init__.py

```diff
@@ -29,3 +39,61 @@ def simulation_vector_description(vector):
         )
 
     return description
+
+
+def si_prefixed(
```

Comment:
Should we consider to move logic out of this __init__ file to separate modules (both simulation_vector_description and si_prefixed)?

Reply:
Yes, that might be a good idea, now that `_abbreviations` is expanding. 👍